deploy/dapr-multi-app/dapr.yaml

```diff
@@ -30,4 +30,20 @@ apps:
   - appID: pubsub-workflow
     appDirPath: ../../pubsub-workflow
     appPort: 3005
-    command: ["dotnet","run", "--DaprHTTPAppPort=3005"]
\ No newline at end of file
+    command: ["dotnet","run", "--DaprHTTPAppPort=3005"]
+  - appID: scheduler-jobs
+    appDirPath: ../../scheduler-jobs
+    appPort: 3006
+    command: [ "go","run", "scheduler-jobs.go"]
```

Comment:
```suggestion
    command: [ "go","run", "scheduler-jobs.go", "--DaprHTTPAppPort=3006"]
```

Reply:
thanks for catching this. I see this is how it operates for those dotnet apps. I didn't look closely enough at this at first. If you look at the dotnet code it is using that throughout the code and `appsettings.json` as config passed in, but this isn't how my go apps work. In looking more at the config for multi-app runs, I believe I just need to set this:

```
daprHTTPPort: <port>
``` 
in the yaml file here. Let me update all of them accordingly